bitsv/network/services/network.py

```diff
@@ -96,20 +97,30 @@ def __init__(self, network):
             self.bitindex3 = MatterCloud(api_key=mattercloud_api_key, network=self.network)
             self.list_of_apis.appendleft(self.bitindex3)
 
+        planaria_token = os.environ.get(PLANARIA_TOKEN_VARNAME, None)
+        if planaria_token and network == 'main':
+            self.planaria = Planaria(token=planaria_token)
+            self.list_of_apis.append(self.planaria)
+
```

Comment:
If I am understanding correct - this will only be added to `self.list_of_apis` if a planaria token is supplied as an env var.

but that means that `PrivateKey.get_bitcom_transactions()` will unexpectedly fail if called without the planaria token right? 

Also Planaria does not implement the `get_balance`, `get_transactions`, `get_transaction`, `get_unspents`, `broadcast_tx` methods so if the NetworkAPI is using Planaria as its "engine" all of these methods will fail...

So a possible solution might be that `Planaria` would be nice to have as a bitsv.network.service that can be instantiated and used like `planaria_instance.get_bitcom.transactions()` but for the `NetworkAPI`, for it to be added, Planaria needs to support all 5 common denominator methods: `get_balance`, `get_transactions`, `get_transaction`, `get_unspents`, `broadcast_tx` and with the exact same normalised return value format...

Reply:
I'm having some trouble replying to you here, I'm sorry.  I tried to copy the way API keys were used in a pre-existing part of the system.  I haven't verified the no-token case; the error may not be clear.